### PR TITLE
Removing old SSZ readme and replacing it with cargo badge.

### PR DESCRIPTION
**Pull request by krsnnik**
## Issue Addressed

#512 

## Proposed Changes

Removing old SSZ readme and replacing it with cargo badge.

### CODE REVIEW

File: eth2/utils/ssz/README.md [hunk 1]
```diff
@@ -1,543 +1,3 @@
...
-
-// Returns the result (Vector of usize) and the index of the next
-let decoded: (Vec<usize>, usize) = decode_ssz_list(&serialized_list, 10).unwrap();
-
-// Check for correctness
-assert_eq!(decoded.0, vec![15,15,15,15]);
-
-assert_eq!(decoded.1, 46);
-```
+eth2_ssz = "0.1.2"
```
Comment (paulhauner) [inline eth2/utils/ssz/README.md]: I was thinking a crates.io "badge", like this one:

Crates.io

```
![Crates.io](https://img.shields.io/crates/v/eth2_ssz)
```

FYI, I made it at https://shields.io/ :)

Comment (paulhauner) [reply eth2/utils/ssz/README.md]: No worries at all, I wasn't very clear :)

**Comment by paulhauner** [comment]
Thank you!

**Comment by blacktemplar** [comment]
Shouldn't this link to the actual crate in crates.io? As it is now the badge links to its image and is very confusing since it only displays the version but not which crate...

**Pull request merged**